### DISCUSSION TITLE
Emojis in Docs not working

### DISCUSSION THREAD

**Discussion by CWempe** [Koenkk/zigbee2mqtt#7228] [General]
I noticed the emojis are not working in the docs.

Example:

https://www.zigbee2mqtt.io/devices/TS0601_thermostat.html

![image](https://user-images.githubusercontent.com/1270359/116308901-61145980-a7a8-11eb-92a7-c93f78984036.png)

https://github.com/Koenkk/zigbee2mqtt.io/blob/master/docs/devices/TS0601_thermostat.md

![image](https://user-images.githubusercontent.com/1270359/116308967-77bab080-a7a8-11eb-80bb-8e9cbe30a0b5.png)

I guess the docs website cannot render the markdown syntax from Github?!

Maybe the emojies can be replaced by the UTF8 code?

But I am no expert.

Is this something that can and should be fixed?
I could create an issue if that is something we want to track.

**Comment by Koenkk** [comment]
Fixed now, thanks!

**Comment by CWempe** [comment]
Great 👍
